### ISSUE TITLE
Is there way to serve swagger ui from swagger file?

### ISSUE THREAD

**Issue by Kavan72** [juhaku/utoipa#599]
I'm looking to just serve swagger ui from `swagger.yaml`, is there way i can do this ? for java we are using [springdoc-openapi](https://springdoc.org/v2/) on this we have [option](https://springdoc.org/v2/#what-is-a-proper-way-to-set-up-swagger-ui-to-use-provided-spec-yml) to enable this.

**Comment by juhaku** [comment]
Yes there is if you are using `rocket`, `axum` or `actix_web` otherwise it is manual procedure. You can use this to expose external openapi spec. https://docs.rs/utoipa-swagger-ui/latest/utoipa_swagger_ui/struct.SwaggerUi.html#method.external_url_unchecked

Note that there is no currently way to serve openapi spec directly from file.

**Comment by Drifter-J** [comment]
@juhaku this was answered so I suggest closing the issue.

**Comment by juhaku** [comment]
Closing due to no activity
